setup/project_template/setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
+
+setuptools.setup(
+    setup_requires=['setuptools-odoo'],
+    odoo_addon=True,
+)
```

Comment:
May this file be removed from the commit?

Reply:
Done :)